### PR TITLE
feat(workspace): globbing support for root patterns

### PR DESCRIPTION
**Pull request by yatli**

### CODE REVIEW

**Comment by chemzqm** [comment]
You should not use glob for this case, use `minimatch` instead when the pattern contains `*`.

**Comment by yatli** [comment]
roger that.

File: src/util/fs.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import os from 'os'
 import path from 'path'
 import readline from 'readline'
 import util from 'util'
+import glob = require('glob')
```
Comment (chemzqm) [inline src/util/fs.ts]: No glob.

Comment (yatli) [reply src/util/fs.ts]: done.

**Comment by yatli** [comment]
@chemzqm please take another look.

**Pull request merged**